pyproject.toml

```diff
@@ -23,8 +23,23 @@ build-verbosity = 1
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "arm64"]
 
+[tool.bandit]
+# Excluding tests because bandit doesn't like `assert`.
+exclude_dirs = ["docs/conf.py", "tests", "install", "conda/cmake-package-test", "tools", "setup.py"]
+
+[tool.codespell]
+ignore-words-list = "elemt"
```

Comment:
Where are we using this?

Reply:
In C++. There are template parameters called `ElemT`. Could rename them if you think ignoring this word is risky.

Reply:
Ok, so it is not case-sensitive?

You can leave it.